src/arrays.js

```diff
@@ -1,59 +1,189 @@
 };
+/*
+'returns the array elements as a comma-seperated string'
+expect(arrayToCSVString(['a', 'b', 'c', 'd'])).toEqual('a,b,c,d')
+expect(arrayToCSVString([1, 2, 3, 4, 5])).toEqual('1,2,3,4,5')
+*/
 
 export const csvStringToArray = (string) => {
-  // your code here
+  return string.split(",");
 };
+/*
+'converts the csv string as an array'
+expect(csvStringToArray('a,b,c,d')).toEqual(['a', 'b', 'c', 'd'])
+expect(csvStringToArray('1,2,3,4,5')).toEqual(['1', '2', '3', '4', '5'])
+*/
 
-export const addToArray = (array, element) => {
-  // your code here
+export const addToArray = (element, array) => {
+  array.push(element);
 };
+/*
+'adds the item to the end of the array'
+const array = []
+const array2 = [1, 2, 3]
+    
+expect(addToArray('a', array)).toEqual(undefined)
+expect(addToArray(4, array2)).toEqual(undefined)
+
+expect(array).toEqual(['a'])
+expect(array2).toEqual([1, 2, 3, 4])
+*/
 
-export const addToArray2 = (array, element) => {
-  // your code here
+export const addToArray2 = (element, array) => {
+  return array.concat(element);
 };
+/*
+'returns a new array with the value appended'
+const array = ['a', 'b', 'c'];
+const array2 = [1, 2, 3];
+
+expect(addToArray2('d', array)).toEqual(['a', 'b', 'c', 'd'])
+expect(array).toEqual(['a', 'b', 'c'])
+
+expect(addToArray2(4, array2)).toEqual([1, 2, 3, 4])
+expect(array2).toEqual([1, 2, 3])
+*/
 
 export const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index, 1);
+  // array.splice(index, howmany)
+  // https://blog.mariusschulz.com/2016/07/16/removing-elements-from-javascript-arrays
 };
+/*
+'removes the element at position n'
+const array = ['ant', 'bison', 'cockerel', 'duck', 'elephant']
+removeNthElement(2, array)
+expect(array).toEqual(['ant', 'bison', 'duck', 'elephant'])
+*/
 
 export const numbersToStrings = (numbers) => {
-  // your code here
+  return numbers.map(String);
 };
+/*
+'converts every number in the array to a string'
+expect(numbersToStrings([1, 2, 3])).toEqual(['1', '2', '3'])
+*/
 
 export const uppercaseWordsInArray = (strings) => {
-  // your code here
+  return strings.map(function(n) {
+    return(n).toUpperCase()
+    });
 };
+/*
+'makes every string in the array uppercase'
+expect(uppercaseWordsInArray(['cat', 'mouse', 'banana'])).toEqual(['CAT', 'MOUSE', 'BANANA'])
+*/
 
 export const reverseWordsInArray = (strings) => {
-  // your code here
+  const animalsRev = strings.map((strings) => {
+    const char = strings.split("");
+    const reverseChar = char.reverse();
+    return reverseChar.join("");
+    })
+  return animalsRev;
+  // return strings.split("").reverse().join("");
+  // http://eddmann.com/posts/ten-ways-to-reverse-a-string-in-javascript/
 };
+/*
+'reverses every string in an array'
+expect(reverseWordsInArray(['cat', 'Mouse', 'banana'])).toEqual(['tac', 'esuoM', 'ananab'])
+*/
 
 export const onlyEven = (numbers) => {
-  // your code here
+  return numbers.filter(function(n){return n % 2===0})
```

Comment:
You could use an arrow function here to make this more concise. Also, space things out a bit and avoid meaningless variable names such as `n`: 
```js
return numbers.filter(number => number % 2 === 0)
```